src/pages/sidebar.page.tsx

```diff
@@ -0,0 +1,4 @@
+import Sidebar from '../component/SideBar'
```

Comment:
一般的に、相対 import だとファイルの位置変更に弱かったり、特に `..` を含むような相対 import だとファイル同士の関係が見ずらいので、絶対 import を使って
```suggestion
import Sidebar from '@/component/SideBar'
```
にしてほしいです :pray: